### ISSUE TITLE
Customizable Vim Bindings

### ISSUE THREAD

**Issue by 0x0D15** [searxng/searxng#214]
Would you consider adding the option to customize the keybindings of the vim plugin? That plugin is fantastic but the lack of any sort of customizability is unfortunate and would be a great addition.

**Comment by pranavsetpal** [comment]
I agree, it is quite irritating to use 'j' and 'k' for me who uses the Colemak layout as they are present in Qwerty's 'y' and 'n' positions respectively. This sort of customization would make the experience, at least for those like me, a lot better!

**Comment by return42** [comment]
There doesn't seem to be much interest in the feature, otherwise someone would have implemented it already.
